tools/question_converters/canvas/canvas.py

```diff
@@ -214,7 +214,7 @@ def students(self):
         return students
 
 
-class CourseSubObject(Canvas):
+class CourseSubObject(Course):
```

Comment:
This inheritance tree makes more sense to me in order to resolve the bug below.

Reply:
There is no client code here, correct?

Reply:
Correct.

Reply:
Did you test this?